iot_config/src/route_service.rs

```diff
@@ -220,13 +290,22 @@ impl iot_config::Route for RouteService {
     ) -> GrpcResult<Self::get_euisStream> {
         let request = request.into_inner();
 
-        let euis = route::list_euis_for_route(&request.route_id, &self.pool)
-            .await
-            .map_err(|_| Status::internal("get euis failed"))?;
+        self.verify_request_signature(&request, OrgId::RouteId(&request.route_id))
+            .await?;
 
+        let pool = self.pool.clone();
         let (tx, rx) = tokio::sync::mpsc::channel(20);
+
         tokio::spawn(async move {
-            for eui in euis {
+            let euis = match route::list_euis_for_route(&request.route_id, &pool)
+                .map_err(|_| Status::internal("failed retrieving eui pairs"))
```

Comment:
Should we be logging these types of errors?

Reply:
Change of plans: we’re returning them to the caller to decide what to do with them. We can also log if we decide we need to